Source/Fuse.Scripting.JavaScript/JavaScript.Model.uno

```diff
@@ -0,0 +1,254 @@
+using Uno;
+using Uno.UX;
+using Uno.Collections;
+using Uno.Compiler;
+using Fuse.Scripting;
+using Uno.Testing;
+using Uno.Threading;
+using Uno.IO;
+
+namespace Fuse.Reactive
+{
+	//the attached properties need to be in a public class.
+	public partial class JavaScript
+	{
+		class ModelData
+		{
+			public IExpression Model;
+			public NameTable NameTable;
+		}
+		
+		//Requires a NameTable that will be set first.
+        [UXAttachedPropertySetter("JavaScript.Model"), UXNameScope, UXAuxNameTable("ModelNameTable")]
+        public static void SetModel(Visual v, IExpression model)
+        {
+			var md = v.Properties.Get( _modelHandle ) as ModelData;
+			if (md == null)
+			{
+				md = new ModelData{ Model = model };
+				v.Properties.Set( _modelHandle, md );
+			}
+			else
+			{
+				md.Model = model;
+			}
+			
+			Complete( md, v );
+		}
+		
+		static void Complete( ModelData md, Visual v )
+		{
+			v.RemoveAllChildren<ModelJavaScript>();
+			
+			//avoid creating without the NameTable as unfortunately UX will set the Model prior to the NameTable
+			if (md.NameTable == null || md.Model == null)
+				return;
+				
+			var parsed = ModelJavaScript.ParseModelExpression(md.Model, md.NameTable);
+			v.Children.Add( new ModelJavaScript(parsed, md.NameTable, null) );
+        }
+
+        static PropertyHandle _modelHandle = Properties.CreateHandle();
+        
+        [UXAttachedPropertySetterAttribute("ModelNameTable")]
+        public static void SetModelNameTable(Visual v, NameTable nt)
+        {
```

Comment:
Broken indentation. Whoever wrote this, fix your config. This keeps happening in this source-file. Fix all of it.